qiskit/circuit/library/standard_gates/t.py

```diff
@@ -89,7 +89,7 @@ class TdgGate(Gate):
 
         Tdg = \begin{pmatrix}
                 1 & 0 \\
-                0 & 1-i
+                0 & e^{i\pi/4}
```

Comment:
I think a similar normalization is needed for the TGate docstring

Reply:
fixed